pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -252,7 +142,14 @@ def _supports_adjoint(circuit):
 
 def _adjoint_ops(op: qml.operation.Operator) -> bool:
     """Specify whether or not an Operator is supported by adjoint differentiation."""
-    return adjoint_ops(op) and not isinstance(op, qml.PauliRot)
+
+    if isinstance(op, (Conditional, MidMeasureMP, PauliRot)):
+        return False
```

Comment:
Shall we prefer the compact version in PL?
```suggestion
    if isinstance(op, (Conditional, MidMeasureMP, PauliRot)): return False
```

Reply:
Will update.